packages/core/src/common/_color-aliases.scss

```diff
@@ -7,7 +7,7 @@ Color aliases
 These variables are semantic aliases of our [colors](#colors). They are used throughout Blueprint
 itself to ensure consistent color usage across components.
 
-<table class=pt-table>
+<table class=pt-html-table>
```

Comment:
missing quotes. though it seems to work.

Reply:
def works. let's add them for good measure